apps/src/StudioApp.js

```diff
@@ -809,7 +809,7 @@ StudioApp.prototype.renderShareFooter_ = function (container) {
       },
       {
         text: window.dashboard.i18n.t('footer.how_it_works'),
-        link: location.href + "/edit",
+        link: location.href + "/view",
```

Comment:
Going with the last option (leave "How it works" as-is) because we're probably scheduling the unification of project URLs as @larrypo's next task.